lib/api/model/narrow.dart

```diff
@@ -47,15 +47,53 @@ class ApiNarrowTopic extends ApiNarrowElement {
 }
 
 /// An [ApiNarrowElement] with the 'dm', or legacy 'pm-with', operator.
+///
+/// An instance directly of this class must not be serialized with [jsonEncode],
+/// and more generally its [operator] getter must not be called.
+/// Instead, call [resolve] and use the object it returns.
 class ApiNarrowDm extends ApiNarrowElement {
-  @override String get operator => 'pm-with'; // TODO(#146): use 'dm' where possible
+  @override String get operator {
```

Comment:
This sort of thing was why I was asking about the dart format.  I see both this style and the more traditional style throughout the codebase.
```
@override
String get operator {}
```

Reply:
Yeah, for the classes in this file (before this PR, anyway), they're all so short and simple and parallel to each other that I felt the one-line form like
```dart
  @override String get operator => 'topic';
```
was helpful for reading, by helping make the parallelism easy to see.

If we did adopt automated formatting, though, I wouldn't really mind these going into multi-line form as a consequence of that.

At a quick grep (`git grep '@override '`), I'm only finding one other spot in the codebase where we use the one-line form:
```dart
class UriConverter extends TypeConverter<Uri, String> {
  const UriConverter();
  @override String toSql(Uri value) => value.toString();
  @override Uri fromSql(String fromDb) => Uri.parse(fromDb);
}
```
The idea there is similar, and again it's not something I'd particularly regret going away if we adopted auto-formatting.